src/main.rs

```diff
@@ -310,6 +366,29 @@ async fn main() -> Result<(), Box<dyn std::error::Error>> {
 
     let num_validators = value_t_or_exit!(matches, "number_of_validators", usize);
     let num_rpc_nodes = value_t_or_exit!(matches, "number_of_rpc_nodes", usize);
+    let client_config = ClientConfig {
+        num_clients: value_t_or_exit!(matches, "number_of_clients", usize),
+        client_type: matches
+            .value_of("client_type")
+            .unwrap_or_default()
```

Comment:
I think with `default_value` you can just do a plain `unwrap`, since `unwrap_or_default` will give an empty string